src/commoncode/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 def set_re_max_cache(max_cache=1000000):
     """
-    Set re and fnmatch _MAXCACHE to 1Million items to cache compiled regex
+    Set re and fnmatch _MAXCACHE to 1 Million items to cache compiled regex
```

Comment:
Again,this is repeation of https://github.com/nexB/scancode-toolkit/pull/1534/commits/ff7612b0da18453ee9b7ff2f552b266043facbd6 .